spec/models/solidus_braintree/gateway_spec.rb

```diff
@@ -220,6 +221,37 @@
           expect(authorize.message).to eq 'authorized'
           expect(authorize.authorization).to be_present
         end
+
+        context 'with available device_data' do
+          it 'passes "fake-device-data" as the device_data parameter in the request' do
```

Comment:
Another nit: using the fixture data in the test description won't read well when it comes to understanding the behavior.

Reply:
You're right. I updated the test descriptions :+1: